exampleSite/content/post/math-typesetting.md

```diff
@@ -9,41 +9,18 @@ math: true
 Mathematical notation in a Hugo project can be enabled by using third party JavaScript libraries.
 <!--more-->
 
-In this example we will be using [KaTeX](https://katex.org/)
+In this example we will be using [MathJax](https://www.mathjax.org/)
 
-- Create a partial under `/layouts/partials/math.html`
-- Within this partial reference the [Auto-render Extension](https://katex.org/docs/autorender.html) or host these scripts locally.
-- Include the partial in your templates like so:  
+- To enable MathJax globally set the parameter `math` to `true` in a project's configuration
```

Comment:
@catap 
Is there a specific reason why we need an explicit configuration to enable MathJax?